### DISCUSSION TITLE
PS 7 get-service parameter missing

### DISCUSSION THREAD

**Discussion by Schutzhund3** [PowerShell/PowerShell#15639] [Q&A] [answered]
I found the -computername parameter is no longer available in PS 7.0.6 or above.  What is the work around for this?

**Comment by vexx32** [comment, marked as answer]
You can use `Invoke-Command` to target the machine and call `Get-Service` on the machine in that way.

**Reply by jimbobmcgee** [reply]
Is that a conscious decision to forego the RPC-based remoting in favour of WinRM/SSH-based remoting?
Because---at least for me---WinRM/SSH has a significant setup overhead, while RPC-based stuff usually *just works*...

**Reply by vexx32** [reply]
I don't recall the exact rationale, but yeah I think the general trajectory is away from RPC for everything.

@SteveL-MSFT might be able to shed more light on the rationale here.

**Reply by iSazonov** [reply]
MSFT strategic decision (not in PowerShell team) was to do not invest in too old protocols (SOAP WinRM based on, WMI/CIM). As result we get amazing regressions in remoting.
